prow/proxy-common.inc

```diff
@@ -47,17 +47,14 @@ fi
 BAZEL_OUT="$(bazel info ${BAZEL_BUILD_ARGS} output_path)/k8-fastbuild/bin"
 export BAZEL_OUT
 
-# Disable RBE execution due to failures like https://prow.istio.io/view/gcs/istio-prow/pr-logs/pull/istio_proxy/2633/release-test_proxy/211
-export BAZEL_BUILD_RBE_JOBS=0
```

Comment:
We need a few larger machines to build V8 using RBE execution, otherwise it timeouts after 60mins.

@howardjohn do you have access to configure RBE worker pool? If not, do you know who has?

@lizan are there any instructions on how to provision and tag `LARGE_MARCHINE` instances for Envoy? I thought it was documented in Envoy, but I cannot find it now.

Reply:
Actually, Prow has beefy enough machines that we shouldn't need RBE execution.

I'll split this change into separate a PR once we have `LARGE_MACHINE` instances configured, since RBE cache is a big improvement by itself.

Reply:
I do have, if you let me know the commands to run I can do it, if still needed